### PR TITLE
Chore: add Accessibility subcategory and resources to Frontend

### LINKED ISSUE #794: [Add] Accessibility resources
### Site URL

[https://www.a11yproject.com/checklist/](https://www.a11yproject.com/checklist/)

### Description of the above provided site

I would like to add multiple resources and I'd like to start by adding an Accessibility sub-category to the Frontend category.

Resources:
- [Accessibility Checklist by The A11y Project](https://www.a11yproject.com/checklist/)
- [WhoCanUse - color checker](https://whocanuse.com/)
- [WAVE Web Accessibility Evaluation Tools](https://wave.webaim.org/)
- [Microsoft Inclusive Design Methodology](https://inclusive.microsoft.design/)

### Category

Frontend

### Sub Category

accessibility

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [ ] I am a GSSoC'23 contributor

### PR DESCRIPTION
**Pull request by ha-anna**
Closes #794 

## Changes proposed

- Added `accessibility.json` to frontend folder
- Added resources to the `accessibility.json` 
- Added ` { name: 'accessibility', url: '/accessibility' }` to `database/data.ts`
- Added `export { default as accessibility } from './frontend/accessibility.json'` to `database/index.ts`

## Screenshots

Screen Shot 2023-05-28 at 7 09 29 PM
Screen Shot 2023-05-28 at 7 09 43 PM
Screen Shot 2023-05-28 at 7 09 20 PM

## Note to reviewers

Hope everything checks out, thanks.

### CODE REVIEW

**Review by rupali-codes — DISMISSED** [review]
lgtm

File: database/frontend/accessibility.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+[
+  {
+    "name": "Accessibility Checklist",
+    "description": "Accessibility Checklist made by The A11y Project. It uses The Web Content Accessibility Guidelines (WCAG) as a reference point, which is a shared standard for web content accessibility for individuals, organizations, and governments.",
```
Comment (CBID2) [inline database/frontend/accessibility.json:4]: [Proofreading]
```suggestion
    "description": “ This checklist uses The Web Content Accessibility Guidelines (WCAG) as a reference point, which is a shared standard for web content accessibility for individuals, organizations, and governments.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @ha-anna! :) I pointed out a proofreading error in my inline comment. In the future, consider running your descriptions through Grammarly or Spellcheck before making a pull request. Other than that, you're off to a great start! :)

**Review by CBID2 — DISMISSED** [review]
LGTM! 😊

**Comment by ha-anna** [comment]
> Hi @ha-anna! :) I pointed out a proofreading error in my inline comment. In the future, consider running your descriptions through Grammarly or Spellcheck before making a pull request. Other than that, you're off to a great start! :)

Oops, my bad! Thank you for fixing that, will make sure to not make the same mistake again. 🙆🏻‍♀️

**Comment by rupali-codes** [comment]
Your vercel deployment is getting failed because of the following errors, could you please resolve them?

image

**Comment by ha-anna** [comment]
> Your vercel deployment is getting failed because of the following errors, could you please resolve them?
> 
> image

Sure, seems something went wrong after the changes were made, I'll look into it!

**Comment by ha-anna** [comment]
@rupali-codes Seems everything is working now!

**Review by rupali-codes — APPROVED** [review]
lgtm

**Comment by rupali-codes** [comment]
> @rupali-codes Seems everything is working now!

Thanks

**Pull request merged**